### ISSUE TITLE
Scale back tips to just be pulse

### ISSUE THREAD

**Issue by karmatosed** [WordPress/gutenberg#9242]
Tips are useful but they can be a little over zealous and at times get in way. Let's scale back them to their essence and make them useful again.

At the start we can have even more than one pulsing ... this is an indicator of something you can click to see information:

![1](https://user-images.githubusercontent.com/253067/44466831-2af6a300-a619-11e8-8ed8-538f8deef2ac.png)

Clicking the pulsing dot then shows the tip:

![2](https://user-images.githubusercontent.com/253067/44466850-32b64780-a619-11e8-8aba-56c09630e048.png)

I have some ideas for design around Tips but let's get the behaviour fixed first. This hopefully opens up having them display in a few places at same time. We may have to adjust copy to allow this later. I think it's fair to say if you click 'next' the next Tip would show not just dot. However on first load just the pulsing dot shows.

**Comment by tofumatt** [comment]
Have you checked to see if users (especially new users, the target demographic here) know to click on the pulsing dot? I'm not sure I would… having a tip appear on `hover` of the dot would make it a bit more discoverable and a bit more likely to be activated while still reducing the current IN YOUR FACE nature of the NUX tips 😄

**Comment by earnjam** [comment]
I have concerns that if you change to only dots and they remain in the current "one tip at a time" format, it might get ignored by novice users. They would be much harder to ignore if more than one dot were visible at the same time, so I'd definitely lean toward that. 

Right now they're sort of in a publishing workflow order (add block > settings > preview > publish). If more than one were visible, and the user clicks into one of the later tips first, what should the "next tip" process do? Would it step back to the beginning of the list? Would it pick up in the middle and work through it from there?

Stepping back a bit, has the common new user feedback been that the tooltips are intrusive and get in the way? I don't find them to be particularly intrusive since they only show up the first time you open Gutenberg, but I do think they could be refined a bit to be more helpful (copy tweaks, a couple additional tips for some of the less-intuitive editor controls, an obvious way to pause tips & resume later, etc).

**Comment by karmatosed** [comment]
I'll add a bit of context as to what is strong motivation here. Currently people are seeing in some cases multiple Tips, this defeats the purpose and dilutes.

> Have you checked to see if users (especially new users, the target demographic here) know to click on the pulsing dot?

Having anything 'have' to be read is a fail, Tips should always be supplementary not requirement. This was their design. We have no evidence in this direct interface of clicking pulsing because we've never had it. What we do have evidence of is overload.

We also by having pulsing can within reason explore more than one at a time - which is a huge step forward and something we should really do. This would yes take away from workflow, I don't see that as a bad thing. Perhaps step one is that and then it leads into cases for multiple, we can explore.

**Comment by youknowriad** [comment]
How can we move this forward? It's targeted for 4.2 and doesn't have any corresponding PR? What's the action items here?

**Comment by noisysocks** [comment]
I'll spend some time on this this week. I don't think it's a technically challenging task, but will require some back-and-forth iteration with @karmatosed 🙂

**Comment by youknowriad** [comment]
Moving out of 4.2, it doesn't seem ready.

**Comment by noisysocks** [comment]
Tammie and I spoke with each other and agreed that we should more holistically revisit the UX of tips rather than go with this direction of scaling tips back to just be the pulsating dot.

@karmatosed: Are you able to coordinate with the incoming design lead to get a new issue opened up?
